operate/client/src/modules/utils/processDefinition.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+
+const getFullyQualifiedProcessDefinitionName = (processName: string, processVersion: any) => {
+  return `${processName} v${processVersion}`;
+};
+
+const getFullyQualifiedProcessDefinitionNameBy = (definition: any) => {
+  if(!definition) return undefined;
+
+  // for now the definition can be of different types, so we try to get the name and version from different properties
```

Comment:
this is a bit ugly and we could split this also into 2-3 methods with explicit types or do some typecheck here. But we thought being pragmatic is good enough here 😅